daft/dataframe/dataframe.py

```diff
@@ -381,7 +528,8 @@ def iter_rows(
             {'foo': 2, 'bar': 'b'}
             {'foo': 3, 'bar': 'c'}
 
-        !!! tip "See also [`df.iter_partitions()`][daft.DataFrame.iter_partitions]: iterator over entire partitions instead of single rows"
+        Tip:
+            "See also [`df.iter_partitions()`][daft.DataFrame.iter_partitions]: iterator over entire partitions instead of single rows"
```

Comment:
```suggestion
            See also [`df.iter_partitions()`][daft.DataFrame.iter_partitions]: iterator over entire partitions instead of single rows
```